spec/cc/yaml/nodes/engine_spec.rb

```diff
@@ -28,4 +28,15 @@
     engine = config.engines["rubocop"]
     engine.enabled?.must_equal false
   end
+
+  specify 'to_json' do
```

Comment:
Hmm, let me think about it. This test is testing `to_json`; it just so happens that the behavior of `to_json` was fixed by implementing an `as_json` method. I think it's OK the way it is.